src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -5511,6 +5511,7 @@ int RGWRados::restore_obj_from_cloud(RGWLCCloudTierCtx& tier_ctx,
     ret = rgw_cloud_tier_get_object(tier_ctx, false,  headers,
                                 &set_mtime, etag, accounted_size,
                                 attrs, &cb);
+    in_progress = false;
```

Comment:
Why are we always setting to false here? Can we use it similarly to the "rgw_cloud_tier_restore_object" api in which api sets the value accordingly?